data/man/tilix

```diff
@@ -75,13 +75,13 @@ Do not show client side decorations and toolbar.
 Do not show window borders.
 .TP
 .B \-\-new\-process
-Start an additional Tilix instance as a new process. By default and as per GTK+ 3 guidelines, Tilix maintains a single process that all windows belong to. This option forces Tilix to start as a separate process. This is not recommended and is intended for debugging purposes only, using this option will prevent different Tilix windows from communicating with each other. 
+Start an additional Tilix instance as a new process. By default and as per GTK3+ guidelines, Tilix maintains a single process that all windows belong to. This option forces Tilix to start as a separate process. This is not recommended and only intended for debugging purposes. Its use will prevent different Tilix windows from communicating with each other. 
```

Comment:
In the [official GTK+ blog](https://blog.gtk.org/), they always use GTK+ [version_number], so maybe it should keep that format?

Reply:
I agree. It isn't GTK3+, it's GTK+ version 3.